packages/identitykit/src/lib/service/transport-builder/new-tab-transport.builder.ts

```diff
@@ -6,6 +6,5 @@ export async function getPopupTransportBuilder(
 ): Promise<Transport> {
   return new PostMessageTransport({
     ...options,
-    statusTimeout: options.statusTimeout || 15000,
```

Comment:
seems to be disconnectTimeout with latest sea snake libs

Reply:
Actually `statusTimeout` is now `establishTimeout`.

Meanwhile `disconnectTimeout` is a new config introduced with heartbeats which is the duration of not receiving heartbeat responses it will take for a channel to be considered disconnected. Previously this config didn't exist, instead there was the `windowCloseMonitoringInterval` that defined how frequently the closed status of the signer window was checked.

Hopefully the new variable names are more clear:
- `establishTimeout`: how long do we wait before we give up trying to establish connection with signer
- `disconnectTimeout`: how long do we wait before we consider a silent connection as disconnected

Reply:
The establishTimeout has been added, thanks!